pyproject.toml

```diff
@@ -1,8 +1,26 @@
 [tool.poetry]
 name = "testsuite"
```

Comment:
what about making this name `buildingsync-testsuite`? thoughts?

Reply:
I like that 👍

Reply:
wait @macintoshpie which one do you like?

Reply:
weird, I had commented on Nick's comment, but it showing after yours?
I don't really have a preference for what we change it to, but changing it from `testsuite` would be good

<img width="727" alt="Screen Shot 2021-03-08 at 12 33 26 PM" src="https://user-images.githubusercontent.com/18518728/110371871-cd75a680-800a-11eb-9504-18d967d50b4d.png">

Reply:
@macintoshpie -- which one you want? We can also leave it and deal with it at another time.  If I had to choose one I would say 1) `buildingsync-testsuite` or 2) `bsync-testsuite`. I don't have an issue with long package names because you can always alias `import buildingsync-super-awesome-testsuite as bsts`

Reply:
i think i'm going to leave it as testsuite for now. During the documentation updates let's decide if we should rename.